src/projects/EnsureThat/Annotations/NotNullAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿// // --------------------------------------------------------------------------------------------------------------------
```

Comment:
Not that I don't think origin's/source's should be attributed but what's in here that requires it?